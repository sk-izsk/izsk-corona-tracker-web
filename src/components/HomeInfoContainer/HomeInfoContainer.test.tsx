import { render } from '@testing-library/react';
import React from 'react';
import { HomeInfoContainer } from '..';

const mockData = {
  name: 'mock name',
  value: 123,
};

test('HomeInfoContainer renders correctly', () => {
  const { asFragment } = render(<HomeInfoContainer {...mockData} />);
  expect(asFragment()).toMatchSnapshot();
});
