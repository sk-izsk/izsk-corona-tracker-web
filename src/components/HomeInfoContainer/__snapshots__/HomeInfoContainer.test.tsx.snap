// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeInfoContainer renders correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-8 makeStyles-mainContainer-1"
  >
    <div
      class="MuiBox-root MuiBox-root-9 makeStyles-halfCircle-2 makeStyles-halfCircle-5"
    />
    <h3
      class="MuiTypography-root makeStyles-header-4 makeStyles-header-7 MuiTypography-h3"
    >
      mock name
    </h3>
    <div
      class="MuiBox-root MuiBox-root-10"
    >
      <div
        class="makeStyles-values-3 makeStyles-values-6"
      >
        0
      </div>
    </div>
  </div>
</DocumentFragment>
`;
